csrankings.csv

```diff
@@ -6793,6 +6793,7 @@ Jingke Li,Portland State University,https://www.pdx.edu/computer-science/jingke-
 Jingkuan Song,UESTC,http://cfm.uestc.edu.cn/~songjingkuan,F5Zy9V4AAAAJ
 Jingling Xue,UNSW,http://www.cse.unsw.edu.au/~jingling,oZlds-AAAAAJ
 Jingqiang Lin,Chinese Academy of Sciences,http://people.ucas.edu.cn/~0010268?language=en,UimjP5sAAAAJ
+Jingqiang Lin,Chinese Academy of Sciences,http://www.escience.cn/people/linjingqiang/index.html,UimjP5sAAAAJ

```

Comment:
There cannot be two entries. Which is correct?

Reply:
I'm sorry，the 6796 item is correct.

Reply:
Please delete the incorrect one.

Reply:
Ok. I have deleted the incorrect one.